esmvaltool/cmorizers/data/utilities.py

```diff
@@ -440,51 +450,52 @@ def fix_bounds(cube, dim_coord):
     return cube
 
 
-def fix_dim_coordnames(cube):
+def fix_dim_coordnames(cube, project_id="OBS6"):
     """Perform a check on dim coordinate names."""
     # first check for CMOR standard coord;
-    for coord in cube.coords():
+    for coord in cube.dim_coords:
```

Comment:
When I developed the change, I had the issue while testing it in E-OBS CMORization runs.

However, since the function is meant to fix dimension coords according to its name, I judged it safer to iterate on dim_coords only since iris API allows it.

In the general case, since the check is made on the axis, before my modification, any aux_coord that happens to be on either X or Y axis would have its metadata replaced by respectively longitude and latitude ones.

Reply:
Hello,
I ran some more testing on this, I didn't encountered problems reversing this line in particular. The unit test passes and my CMORization runs don't run into errors.

Maybe @schlunma the original writer of this function may tell us more about its original intent ? In particular if sub-setting on dimension coords is preferable or not.

Reply:
Sorry, I didn't write this function. I guess I appear in `git blame` since I moved it within the file in https://github.com/ESMValGroup/ESMValTool/commit/4d0165c0f50080759f9fe875dda45de4ee6ac425.